volatility3/framework/symbols/linux/extensions/__init__.py

```diff
@@ -92,53 +92,94 @@ def get_sections(self):
         else:
             num_sects = self._get_sect_count(self.sect_attrs.grp)
         arr = self._context.object(
-            self.get_symbol_table().name + constants.BANG + "array",
+            self.get_symbol_table_name() + constants.BANG + "array",
             layer_name=self.vol.layer_name,
             offset=self.sect_attrs.attrs.vol.offset,
             subtype=self._context.symbol_space.get_type(
-                self.get_symbol_table().name + constants.BANG + "module_sect_attr"
+                self.get_symbol_table_name() + constants.BANG + "module_sect_attr"
             ),
             count=num_sects,
         )
 
         for attr in arr:
             yield attr
 
-    def get_symbols(self):
-        if symbols.symbol_table_is_64bit(self._context, self.get_symbol_table().name):
-            prefix = "Elf64_"
-        else:
-            prefix = "Elf32_"
+    def get_elf_table_name(self):
         elf_table_name = intermed.IntermediateSymbolTable.create(
-            self.context,
-            self.config_path,
+            self._context,
+            "config_name_elf_symbol_table",
```

Comment:
I'd ditch the `config_name` at the start of this.  It's in the configuration, so the `config_name` is kind of implicit/seems a little odd...

Reply:
As I'm not sure to fully understand what you want me to do with the config_name, I'd prefer if you could change it later.

Reply:
Just change it from `"config_name_elf_symbol_table",` to `"elf_symbol_table",`

Reply:
Oh, I thought you were talking about the field type, not the string. Ok this makes sense indeed !